k8s/base/deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: shiny-deploy
+  labels:
+    app: shiny
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: shiny
+  template:
+    metadata:
+      labels:
+        app: shiny
+    spec:
+      initContainers:
+        - name: init-shiny
+          image: busybox:1.28
+          command: ["sh", "-c", "mkdir -p /shiny/logs /shiny/apps"]
+          volumeMounts:
+            - name: shiny-data
+              mountPath: /shiny
+      containers:
+        - name: shiny
+          image: absternator/shiny-server:dev # todo: change to actual one!!! mrcide-
```

Comment:
What changes did you make here? Can we get the image build script/Dockerfile here and building on BK?

Reply:
BK=BuildKite, where we have some non-gha CI builds running,